src/jquery.i18n.emitter.js

```diff
@@ -128,7 +128,7 @@
 		 *  language.
 		 */
 		plural: function ( nodes ) {
-			var count = parseFloat( this.language.convertNumber( nodes[ 0 ], 10 ) ),
+			var count = this.language.convertNumber( nodes[ 0 ], true ),
```

Comment:
Do you want to keep `parseFloat()` just in case `convertNumber()` is overridden by a misbehaving implementation?